shared.js

```diff
@@ -286,18 +292,24 @@ export function BlockBlueVerified(user, headers) {
 	else if (options.blockNftAvatars && user.has_nft_avatar) {
 		if (
 			// group for block-following option
-			!options.blockFollowing && (user.legacy.following || user.super_following)
+			!options.blockFollowing && (user.legacy?.following || user.super_following)
 		) {
-			console.log(logstr, `did not block user with NFT avatar ${user.legacy.name} (@${user.legacy.screen_name}) because you follow them.`);
+			console.log(logstr, `did not block user with NFT avatar ${formattedUserName} because you follow them.`);
 		}
 		else if (
 			// group for block-followers option
-			!options.blockFollowers && user.legacy.followed_by
+			!options.blockFollowers && user.legacy?.followed_by
 		) {
-			console.log(logstr, `did not block user with NFT avatar ${user.legacy.name} (@${user.legacy.screen_name}) because they follow you.`);
+			console.log(logstr, `did not block user with NFT avatar ${formattedUserName} because they follow you.`);
 		}
 		else {
 			QueueBlockUser(user, String(user.rest_id), headers, ReasonNftAvatar);
 		}
 	}
+
+	function formatLegacyName(user) {
+		const legacyName = user.legacy?.name;
+		const screenName = user.legacy?.screenName;
```

Comment:
I think this object is wrong, I think it's `screen_name` because it's json formatting